.github/workflows/cicd.yml

```diff
@@ -157,7 +159,7 @@ jobs:
         run: yarn test
 
       - name: "Authenticate with GCP"
-        if: github.ref == 'refs/heads/main' || (github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'synchronize' || github.event.action == 'reopened' || github.event.action == 'labeled' || github.event.action == 'unlabeled'))
```

Comment:
Do we need to update this step? i'm not clear if it's required for the login public registry later